packages/overlay/test/focus-trap.test.js

```diff
@@ -170,326 +79,55 @@ describe('focus-trap', () => {
-        );
-      });
-
-      it('should focus the overlay part on open', async () => {
-        overlay.opened = true;
-        await oneEvent(overlay, 'vaadin-overlay-open');
-        expect(getFocusedElementIndex()).to.eql(0);
-      });
-
-      it('should not focus the overlay part if the content is already focused', async () => {
-        overlay.opened = true;
-        // Needs to happen after opened and before focus-trap loop is executed
-        focusableElements[1].focus();
-        await oneEvent(overlay, 'vaadin-overlay-open');
-        expect(getFocusedElementIndex()).not.to.eql(0);
-      });
-
-      it('should focus first element with tabIndex=1', async () => {
-        // It's an arguable behaviour, probably overlay should be focused instead
-        focusableElements[1].tabIndex = 1;
-        overlay.opened = true;
-        await oneEvent(overlay, 'vaadin-overlay-open');
-        const idx = getFocusedElementIndex();
-        expect(focusableElements[idx].tabIndex).to.eql(1);
-      });
-
-      it('`_cycleTab` should visit elements inside shadow content on keyboard `tab` actions', async () => {
-        overlay.opened = true;
-        await oneEvent(overlay, 'vaadin-overlay-open');
-
-        // TAB
-        for (let i = 0; i < focusableElements.length; i++) {
-          // Emulating control-state-mixin moving focus
-          if (focusableElements.filter(isElementFocused).length > 1) {
-            i += 1;
-          }
-
-          const focusedIndex = getFocusedElementIndex();
-          expect(focusedIndex).to.equal(i);
-          tabKeyDown(focusableElements[focusedIndex]);
-        }
-        expect(getFocusedElementIndex()).to.eql(0);
-
-        // SHIFT+TAB
-        tabKeyDown(focusableElements[getFocusedElementIndex()], ['shift']);
-        for (let i = focusableElements.length - 1; i >= 0; i--) {
-          expect(getFocusedElementIndex()).to.eql(i);
-
-          // Emulating control-state-mixin moving focus
-          if (focusableElements.filter(isElementFocused).length > 1) {
-            i -= 1;
-          }
-
-          tabKeyDown(focusableElements[getFocusedElementIndex()], ['shift']);
-        }
-        expect(getFocusedElementIndex()).to.eql(focusableElements.length - 1);
-      });
-    });
-
-    describe('empty content', () => {
-      beforeEach(() => {
-        overlay = parent.$.emptyOverlay;
-        overlayPart = overlay.$.overlay;
-        focusableElements = getFocusableElements(overlay.$.overlay);
-      });
-
-      it('should focus the overlayPart when focusTrap = true', async () => {
-        overlay.opened = true;
-        await oneEvent(overlay, 'vaadin-overlay-open');
-        expect(focusableElements[0]).to.be.eql(overlayPart);
-        expect(getFocusedElementIndex()).to.eql(0);
-      });
-    });
   });
 
-  describe('in the global scope', () => {
+  describe('empty', () => {
     beforeEach(async () => {
-      parent = fixtureSync(`
-        <div>
```

Comment:
We don't need to wrap `vaadin-overlay` in `div` because the `fixtureSync` helper creates it under the hood.
The separate wrapper was most likely needed in the past when we were using `<test-fixture>` instead.